src/lib/indexer/index.ts

```diff
@@ -237,6 +237,17 @@ export default class Indexer {
           }
         }
       }
+      if (changeSet.match2Comments) {
+        for (const [, comment] of changeSet.match2Comments) {
+          if (comment.type === 'insert') {
+            const { type, ...record } = comment
+            await db.insertMatch2Comment(record)
+          } else {
+            const { type, ...record } = comment
+            await db.updateMatch2CommentForTransaction(record.transaction_id, record)
+          }
+        }
+      }
```

Comment:
Do we need this?

Reply:
yes

Reply:
marking as resolved as it was lack of understanding what this does.